infrastructure/aat.tfvars

```diff
@@ -13,3 +13,5 @@ database_sku_name = "GP_Gen5_8"
 database_sku_capacity = "8"
 
 http_client_read_timeout = "180000"
+
+hazelcast_health_monitoring_level = "NOISY"
```

Comment:
Do we really want noisy? That will log all the time, not just when we're over a threshold.

Reply:
I need this to be noisy for now on AAT because can help investigate caching issues on AAT.